devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -28,7 +28,12 @@ metadata {
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "0003, 000A, 0019", manufacturer: "Jasco Products", model: "45853", deviceJoinName: "GE ZigBee Plug-In Switch"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "000A, 0019", manufacturer: "Jasco Products", model: "45856", deviceJoinName: "GE ZigBee In-Wall Switch"
     }
-
+    
+	preferences {
+		input title: "Power Correction", description: "Some sensors report the power value as a multiple of ten.  If your device does this you can add a divisor to offset the value.  A value of 1 makes no correction, a value of 10 will divide the sensors value by ten, etc.  If not entered the device will default to 10.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+		input "powerDivisor", "number", title: "Divisor", description: "Divide power value by this number", range: "1..100", displayDuringSetup: false
```

Comment:
the three lines above this are tabbed

maybe your editor has a "convert tabs to spaces" function?